raiden-cli/raspberrypi.md

```diff
@@ -0,0 +1,92 @@
+# Running raiden-cli on the raspberry pi
```

Comment:
I'd spell it **Raspberry Pi** everywhere, that's how they use it on their website

Reply:
done